packages/atomic/src/components/commerce/facets/atomic-commerce-facets/atomic-commerce-facets.tsx

```diff
@@ -104,6 +104,7 @@ export class AtomicCommerceFacets implements InitializableComponent<Bindings> {
             summary: this.summary,
             facet: facet as T,
             field: facet.state.field,
+            key: facet.state.facetId,
```

Comment:
As mentioned in the Jira description (https://coveord.atlassian.net/browse/KIT-3366), this is the actual fix. The remaining changes in this PR are mostly for the sake of uniformity.